MH-FiorellaTheme5/source/stylesheets/styles.css

```diff
@@ -31,17 +31,75 @@ p{
   padding: 0;
   width: auto;
 }
+/* ------ promotion ------ */
+.p-prom-description, .p-prom-price{
+  margin-top: 0px;
+  margin-bottom: 0px;
+  text-align: left;
+  letter-spacing: 3px;
+}
 
-.promotion {
-  align-items: center;
-  background-color: #F3F9EF;
-  height: 300px;
-  margin:0;
-  padding: 0;
-  width: auto;
+.p-prom-price{
+  color: #E83671;
+}
+.prom-price{
+  font-family: 'MerriweatherSans-Bold';
+}
+.title-icon {
+  display: block;
+  overflow: hidden;
+  position: absolute;
+  top: 25px;
+  width: 100%;
+}
+
+.title-icon:before, .title-icon:after {
+  content: "";
+  display: inline-block;
+  vertical-align: middle;
+  position: relative;
+  width: 100%;
+  border-top-style: solid;
+  border-top-width: 1px;
+  border-color: #E83671; 
 }
 
+.title-icon:before {
+  right: 5px;
+  margin-left: -80%;
+}
+.title-icon:after {
+  left: 5px;
+  margin-right: -50%;
+}
+
+.title-icon-img {
+  width: 40px;
+  vertical-align: middle;
+}
+
+.promotion{
+  padding: 10px;
+}
+
+.box-promotions{
```

Comment:
1. Please reorder the properties in alphabetical order
2. Missing Space before `{`